src/operators/gdalsource.md

```diff
@@ -0,0 +1,32 @@
+# GdalSource
+
+The `GdalSource` is a _source operator_ that reads raster data using GDAL.
+The counter part for vector data is the `OgrSource`.
```

Comment:
You could link to the `OgrSource`.

Moreover, could you replace the currently empty pages with placeholders?
From https://rust-lang.github.io/mdBook/format/summary.html you can create a draft chapter by just linking to no page.

It is not really part of this PR, but otherwise you link to all empty pages.

Reply:
79b36d69847bd722e348825fe0903e7cb05ee27d